ts.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  plugins: [
+    "@typescript-eslint"
+  ],
+  extends: [
+    "plugin:react/recommended",
+    "airbnb",
+    "prettier",
+    "plugin:@typescript-eslint/recommended",
+    "prettier/@typescript-eslint",
+    "./rules/prettier.js",
+    "./rules/eslint.js",
+    "./rules/import.js",
+    "./rules/promise.js",
+    "./rules/react.js",
+    "./rules/react-hooks.js"
+  ],
+  rules: {
+  },
+  settings: {
+    react: {
+      pragma: "React",
+      version: "detect"
+    },
+    "import/resolver": {
+      node: {
```

Comment:
What is it needed for?

Reply:
We need this to specify the extension of the files to be parsed by default it is `['js', 'jsx']`
https://github.com/benmosher/eslint-plugin-import#settings